integration_tests/cairo_zero_hint_tests/search_sorted_lower.small.cairo

```diff
@@ -0,0 +1,26 @@
+// The content of this file has been borrowed from LambdaClass Cairo VM in Rust
+// See https://github.com/lambdaclass/cairo-vm/blob/5d1181185a976c77956aaa4247846babd4d0e2df/cairo_programs/search_sorted_lower.cairo
+
+%builtins range_check
+from starkware.cairo.common.find_element import search_sorted_lower
+from starkware.cairo.common.alloc import alloc
+
+struct MyStruct {
+    a: felt,
+    b: felt,
+}
+
+func main{range_check_ptr}() -> () {
+    // Create an array with MyStruct elements (1,2), (3,4), (5,6).
+    alloc_locals;
+    let (local array_ptr: MyStruct*) = alloc();
+    assert array_ptr[0] = MyStruct(a=1, b=2);
+    assert array_ptr[1] = MyStruct(a=3, b=4);
+    assert array_ptr[2] = MyStruct(a=5, b=6);
```

Comment:
Can you also add an element with the first field equal to 3. Since the functionality of the hint is : 

`returns the pointer to the first element in the array whose first field is at least key` . 

Right now the tests are not checking the `first element` functionality.

Reply:
indeed nice thanks will do that!